aip/general/0121.md

```diff
@@ -171,6 +171,7 @@ and in turn do not increase resource management complexity.
 [list]: ./0132.md
 [management plane]: ./0111.md#management-plane
 [output only]: ./0203.md#output-only
+[Output only]: ./0203.md#output-only
```

Comment:
I believe these are case insensitive, so you may remove this - the upper case form should still link properly

Reply:
Done.